app/components/Entity/components/ConfigureWorkflowInputs/components/Main/components/Stepper/components/Step/SequencingStep/components/UploadMyData/uploadMyData.tsx

```diff
@@ -4,11 +4,11 @@ import { Typography } from "@mui/material";
 import { TYPOGRAPHY_PROPS } from "@databiosphere/findable-ui/lib/styles/common/mui/typography";
 import { PAPER_PROPS } from "./constants";
 import { useEffect } from "react";
-import { STEP } from "../../step";
+import { PAIRED_END_STEP } from "../../step";
```

Comment:
this has been a lot to digest so maybe i didnt read it right, but this needs to be a bit cleverer no? or it thinks its not configured for single end workflows?